app/controllers/devise/registrations_controller.rb

```diff
@@ -147,4 +144,25 @@ def account_update_params
   def translation_scope
     'devise.registrations'
   end
+
+  private
```

Comment:
Is this required to be `private`? Other methods above are `protected`. I'm subclassing and overriding some methods of `Devise::RegistrationsController` and need to copy these new private methods during the gem version upgrade.